fusor-ember-cli/app/utils/validators.js

```diff
@@ -235,6 +235,20 @@ const IpAddressValidator = RegExpValidator.extend({
   message: 'This is an invalid ip address.'
 });
 
+const IpTypoValidator = Validator.extend({
```

Comment:
@cfchase, @eriknelson, it helped when I just inherited from `Validator` rather than from `RegExpValidator` which I tried yesterday.